bentoml/deployment/aws_lambda/__init__.py

```diff
@@ -247,18 +252,6 @@ def _apply(
                     deployment_spec.bento_name,
                     deployment_spec.bento_version,
                 )
-                logger.debug(
```

Comment:
as we talked about removing artifacts_prefix offline - just remove the entire `if bento_service_metadata.artifacts:` section?